tox.ini

```diff
@@ -99,3 +111,15 @@ deps =
 commands =
     pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} \
         --durations=0 --asyncio-mode=auto -m legacy_relation
+
+[testenv:bundle-integration]
+description = Run integration tests for legacy relations
+deps =
+    pytest
+    juju
+    psycopg2-binary
+    pytest-operator>0.17.0
```

Comment:
Out of curiosity, what are we pinning the version here for?

Reply:
Not actually sure, I don't think this has been relevant for a while.